src/callbacks_step/amr.jl

```diff
@@ -124,6 +128,17 @@ function initialize!(cb::DiscreteCallback{Condition,Affect!}, u, t, integrator)
   amr_callback = cb.affect!
   semi = integrator.p
 
+  mesh, _, _, _ = mesh_equations_solver_cache(semi)
+
+  if mesh isa P4estMesh
+    if cb.affect!.p4est_partition_allow_for_coarsening != mesh.p4est_partition_allow_for_coarsening
+      if mpi_isroot()
+        @info "The attribute p4est_partition_allow_for_coarsening from the mesh is changed."
```

Comment:
```suggestion
        @info "The attribute p4est_partition_allow_for_coarsening from the mesh is changed to `$(cb.affect!.p4est_partition_allow_for_coarsening)`."
```

Reply:
We should also adjust the corresponding part in  `ignore_content` from [test/test_trixi.jl](https://github.com/trixi-framework/Trixi.jl/pull/1342/files#diff-57294bc80feea88f5712e89ebe10219b5867895c83f66952141a18f75393a476).

Reply:
Good point, thanks!

Reply:
Done!